asllib/doc/ASLTypingReference.tex

```diff
@@ -595,12 +843,20 @@ \section{TypingRule.Subtype}
   \item \texttt{id2} is a subtype of \texttt{id1}.
   \end{itemize}
 
-  This aims to encompass LRM Section 4.3.1 R\_NXRX, I\_KGKS, I\_MTML, I\_JVRM, I\_CHMP.
+  This aims to encompass LRM Section 4.3.1 R\_NXRX, I\_KGKS, I\_MTML, I\_JVRM 
+\todocomment{  
+ROMAN: I don't see how, in the code.
+}
+, I\_CHMP.
 
 \section{TypingRule.Supertype}
 
   \subsection{Prose}
   \texttt{T} is a supertype of \texttt{S} if and only if \texttt{S} is a subtype of \texttt{T}.
+  
+\todocomment{
+ROMAN: This rule doesn't exist in the code. Also, it is redundant, since it just applies to the same arguments in reverse order. I recommend removing this section.
```

Comment:
that is fine - if you do please go ahead and modify the text which uses this notion